internal/oapispec/openapi3.go

```diff
@@ -155,20 +155,20 @@ func ffTagHandler(ctx context.Context, route *Route, name string, tag reflect.St
 
 func addCustomType(t reflect.Type, schema *openapi3.Schema) {
 	typeString := "string"
-	if schema.Type == "" {
-		switch t.Name() {
-		case "UUID":
-			schema.Type = typeString
-			schema.Format = "uuid"
-		case "FFTime":
-			schema.Type = typeString
-			schema.Format = "date-time"
-		case "Bytes32":
-			schema.Type = typeString
-			schema.Format = "byte"
-		case "FFBigInt":
-			schema.Type = typeString
-		}
+	switch t.Name() {
+	case "UUID":
+		schema.Type = typeString
+		schema.Format = "uuid"
+	case "FFTime":
+		schema.Type = typeString
+		schema.Format = "date-time"
+	case "Bytes32":
+		schema.Type = typeString
+		schema.Format = "byte"
+	case "FFBigInt":
+		schema.Type = typeString
+	case "JSONAny":
+		schema.Type = ""
```

Comment:
Should this not be: https://swagger.io/docs/specification/data-models/data-types/#any

Reply:
Seems appropriate - I'll look into it

Reply:
I think this is actually what I'm doing. That link says to omit the type if you want to allow "any type". As a reference, for something like a contract `location`, this is what is written (notice the omission of "type" similar to the examples in the docs):

```
location:
  description: If this API is tied to an individual instance of
    a smart contract, this field can include a blockchain specific
    contract identifier. For example an Ethereum contract address,
    or a Fabric chaincode name and channel
```

Reply:
Thanks @awrichar - makes sense 👍